lib/mutations/date_filter.rb

```diff
@@ -1,21 +1,26 @@
 module Mutations
   class DateFilter < AdditionalFilter
     @default_options = {
-      :nils => false,       # true allows an explicit nil to be valid. Overrides any other options
-      :format => nil,       # If nil, Date.parse will be used for coercion. If something like "%Y-%m-%d", Date.strptime is used
-      :after => nil,        # A date object, representing the minimum date allowed, inclusive
-      :before => nil        # A date object, representing the maximum date allowed, inclusive
+      :nils => false,         # true allows an explicit nil to be valid. Overrides any other options
+      :empty_is_nil => false, # false disallows "".  true allows "" and overrides any other validations (b/c they couldn't be true if it's empty)
```

Comment:
This comment is no longer accurate; it still describes that `:empty` would have had.

Reply:
Fixed - copied the description from `IntegerFilter`